index.bs

```diff
@@ -116,38 +153,65 @@ While a recent version of Edge identifies itself as:
-developers that have stymied historical approaches:
+There's quite a bit of information packed into those strings (along with a fair number of lies 
+which are ignored in practice). 
+
+Use cases for the information contained in `User-Agent` field values include:
 
 1.  Brand and version information (e.g. "Chrome 69") allows websites to work around known bugs in
     specific releases that aren't otherwise detectable. For example, implementations of Content
     Security Policy have varied wildly between vendors, and it's difficult to know what policy to
     send in an HTTP response without knowing what browser is responsible for its parsing and
     execution.
 
-2.  Developers will often negotiate what content to send based on the user agent and platform. Some
+2.  Developers will often negotiate the content to send based on the user agent and platform. Some
     application frameworks, for instance, will style an application on iOS differently from the same
     application on Android in order to match each platform's aesthetic and design patterns.
 
 3.  Similarly to #1, OS revisions and architecture can be responsible for specific bugs which can
-    be worked around in website's code, and narrowly useful for things like selecting appropriate
-    executables for download (32 vs 64 bit, ARM vs Intel, etc).
+    be worked around in website's code, and useful for things like selecting the appropriate 
+    executable for download (32 vs 64 bit, ARM vs Intel, etc).
 
 4.  Sophisticated developers use model/make to tailor their sites to the capabilities of the
     device (e.g. [[FacebookYearClass]]) and to pinpoint performance bugs and regressions which
     sometimes are specific to model/make.
-
-This document proposes a mechanism which might allow user agents to be a bit more aggressive about
-removing entropy from the `User-Agent` string generally by giving servers that really need some
-specific details about the client the ability to opt-into receiving them. It introduces four new
-Client Hints ([[I-D.ietf-httpbis-client-hints]]) that can provide the client's branding and version
-information, the underlying operating system's branding and major version, as well as details about
-the underlying device. Rather than broadcasting this data to everyone, all the time, user agents can
-make reasonable decisions about how to respond to given sites' requests for more granular data,
-reducing the passive fingerprinting surface area exposed to the network.
+    
+5.  Difference models of device perform very differently [[AlexRussell]]. Developers wishing to 
+    improve performance require this insight in analytics tools and in real time to optimise their
+    web site. The ability to optimise on first request is particularly important to authors targeting
+    people living in emerging economies where lower end device models and features are more widely
```

Comment:
What is a "lower end device feature"? Or am I mis-reading this?

Also, can this be re-worded to avoid any western vs non-western distinction? I think focusing on "lower end" vs "higher end", or "emerging markets / economies vs developed markets / economies" is sufficient to convey the meaning (and we don't have to worry about obvious counter-examples like Canada vs South Korea, etc.)